packages/loader/container-loader/src/containerContext.ts

```diff
@@ -279,19 +270,51 @@ export class ContainerContext implements IContainerContext {
         }
 
         for (const comparer of comparers) {
-            const satisfies = await comparer.satisfies(this.codeDetails, constraintCodeDetails);
+            const satisfies = await comparer.satisfies(
+                moduleWithDetails.details,
+                constraintCodeDetails,
+            );
             if (satisfies === false) {
                 return false;
             }
         }
         return true;
     }
 
+    // #region private
+    private attachListener() {
```

Comment:
in the future please don't mix un related refatoring with changes, it just makes the PR harder to review

Reply:
Sure thing. Although, it's somewhat related. Needed to organize new private methods in one place.